Workbooks/Images/Logos/Alsid.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 477.51 515.81"><defs><style>.cls-1{fill:#1b2b44;}.cls-2{fill:#0d06b9;}</style></defs><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path class="cls-1" d="M392.68,178.06l-74-128.31a89.08,89.08,0,0,0-159.85,0L84.75,178.24a91.06,91.06,0,0,0-15.43,50.07,89.14,89.14,0,0,0,89.05,89.05H319.16a89.05,89.05,0,0,0,73.52-139.3Zm-73.52,122H158.37a71.82,71.82,0,0,1-71.75-71.75A73.58,73.58,0,0,1,99.3,187.59c0-.06.06-.13.1-.19l.23-.33L174,58.12c.06-.1.09-.21.14-.31s.1-.14.14-.22a71.78,71.78,0,0,1,129,0c.05.1.12.19.17.29a2.56,2.56,0,0,0,.11.24l74.35,129c.1.16.22.32.33.48l0,.1a71.75,71.75,0,0,1-59.12,112.41Z"/><path class="cls-2" d="M274.59,76.05c-2-3.43-12.72-20.69-35.85-20.69-19.55,0-31.39,12.15-36.65,22.27-4.62,8.9-61.68,107.59-70.68,122.7-5.12,8.59-12.3,24.56-2.58,43.65,8.72,17.15,26.18,21.91,36.52,21.91h144c8,0,23.88-.29,36.64-17.09,13.21-17.4,6.84-37.05,3.88-42.07S276.57,79.48,274.59,76.05ZM332.2,238.34c-7.19,9.47-14.91,10.25-22.86,10.25h-144c-1.51,0-14.94-.35-21.1-12.46-4.47-8.78-3.84-17.09,2-26.94,8.48-14.25,66.18-114,71.17-123.59a24,24,0,0,1,42.17-.9c.51.89,5.57,9.66,12.83,22.29C293.32,143.25,332,210.46,334.89,215.4,335.2,216.08,340.28,227.7,332.2,238.34Z"/><path class="cls-1" d="M156.3,472.57V418.51a4.13,4.13,0,0,0-4.13-4.13H150.4a10.33,10.33,0,0,0-10.33,10.33V474.6c0,29.08,15.05,41.21,48,41.21h28.33a2.58,2.58,0,0,0,2.58-2.59v-13a1,1,0,0,0-1-1H188C164.57,499.21,156.3,491.17,156.3,472.57Z"/><path class="cls-1" d="M322.77,515.34h14.17a1,1,0,0,0,1-1V424.58a10.32,10.32,0,0,0-10.32-10.33h-3.33a2.58,2.58,0,0,0-2.58,2.58V514.3A1,1,0,0,0,322.77,515.34Z"/><path class="cls-1" d="M430.29,414.38c-1.1-.07-2.2-.11-3.32-.11s-2.23,0-3.33.11H363.18a5.16,5.16,0,0,0-5.17,5.17V514.3a1,1,0,0,0,1,1h13.83a1,1,0,0,0,1-1V430.05l.34.44v-.32h51.17v.14c.51,0,1,0,1.55,0a34.55,34.55,0,1,1,0,69.09c-.52,0-1,0-1.55,0v.07h-28.9a2.57,2.57,0,0,0-2.58,2.58v3a10.33,10.33,0,0,0,10.32,10.33h21.15l1.56,0,1.56,0a50.55,50.55,0,0,0,1.76-101Z"/><path class="cls-1" d="M90.49,427.78a26.9,26.9,0,0,0-46.66,0L1.48,501.11l.05,0a9.24,9.24,0,0,0,6.68,14.27,8.77,8.77,0,0,0,1,.06,9.27,9.27,0,0,0,2.84-.45c.24-.07.47-.16.7-.25l.34-.16,68-26.84a2.58,2.58,0,0,0,1.28-3.69h0a15,15,0,0,0-18.47-6.43L23.65,493.6l33.51-58a11.33,11.33,0,0,1,9.93-5.73h.15a11.33,11.33,0,0,1,9.92,5.73l46,79.69,0,.1h10.91a4.13,4.13,0,0,0,3.57-6.2L90.49,427.78Z"/><path class="cls-1" d="M308.47,487c0-18.15-11.43-26.77-38-28.92l-47.11-3.79c-18.11-1.37-22.18-6.12-22.18-12.64,0-7.53,5.61-11.19,17.16-11.19h73a10.33,10.33,0,0,0,10.33-10.33v-3.35a2.59,2.59,0,0,0-2.59-2.59H219.77c-23.72,0-36.79,9.91-36.79,27.9,0,17.08,11.94,26,36,27.92L267.79,474c15.55,1.24,22.23,5.63,22.23,13.64,0,8.39-6.64,11.55-19.21,11.55H243.14A4.12,4.12,0,0,0,239,503.3v11.47a1,1,0,0,0,1,1h31.64c23.73,0,36.79-9.91,36.79-27.9,0-.08,0-.15,0-.23S308.47,487.25,308.47,487Z"/></g></g></svg>
```

Comment:
The logo id attribute has to be a unique id. Could you update:

id="Layer_2"  -->  id="\<generated guid>"
id="Layer_1-2" -->  id="\<generated guid>"